<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialForm</class>
 <widget class="QWidget" name="SerialForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_ctrl">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_SerialPort" stretch="2,5,1">
         <item>
          <widget class="QLabel" name="label_SerialPort">
           <property name="text">
            <string>串口:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_SerialPort"/>
         </item>
         <item>
          <widget class="QPushButton" name="button_SerialPort">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#button_SerialPort::pressed{border-image: url(:/assert/ICON/076.png);}
QPushButton#button_SerialPort{border-image: url(:/assert/ICON/026.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_SerialRate" stretch="2,6">
         <item>
          <widget class="QLabel" name="label_SerialRate">
           <property name="text">
            <string>波特率:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_SerialRate">
           <property name="currentText">
            <string>9600</string>
           </property>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>230400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>460800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>576000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>921600</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_DataWidth" stretch="2,6">
         <item>
          <widget class="QLabel" name="label_DataWidth">
           <property name="text">
            <string>数据位:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_DataWidth">
           <property name="currentText">
            <string>8</string>
           </property>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_StopWidth" stretch="2,6">
         <item>
          <widget class="QLabel" name="label_StopWidth">
           <property name="text">
            <string>停止位:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_StopWidth">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_SerialStatus">
         <item>
          <widget class="QLabel" name="label_SerialStatus">
           <property name="text">
            <string>串口状态:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SerialStatusBack">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/assert/ICON/102.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_button">
         <item>
          <widget class="QPushButton" name="button_OpenSerial">
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_CloseSerial">
           <property name="text">
            <string>关闭串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_ASCII">
           <property name="text">
            <string>ASCII</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_HEX">
           <property name="text">
            <string>十六进制</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_show">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_showText" stretch="9,2,1">
         <item>
          <widget class="QTextEdit" name="textEdit_Text">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_SendHistory"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_inputCtrl">
           <item>
            <widget class="QPushButton" name="pushButton_SendText">
             <property name="text">
              <string>发送数据</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Text"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_SerialPort</sender>
   <signal>clicked()</signal>
   <receiver>SerialForm</receiver>
   <slot>button_SerialPort_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_SendText</sender>
   <signal>clicked()</signal>
   <receiver>SerialForm</receiver>
   <slot>pushButton_SendText_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Text</sender>
   <signal>returnPressed()</signal>
   <receiver>SerialForm</receiver>
   <slot>pushButton_SendText_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_SendHistory</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SerialForm</receiver>
   <slot>listWidget_SendHistory_onClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_CloseSerial</sender>
   <signal>clicked()</signal>
   <receiver>SerialForm</receiver>
   <slot>button_CloseSerial_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_OpenSerial</sender>
   <signal>clicked()</signal>
   <receiver>SerialForm</receiver>
   <slot>button_OpenSerial_onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_SerialPort_onClick()</slot>
  <slot>pushButton_SendText_onClick()</slot>
  <slot>listWidget_SendHistory_onClick(QModelIndex)</slot>
  <slot>button_CloseSerial_onClick()</slot>
  <slot>button_OpenSerial_onClick()</slot>
 </slots>
</ui>
